stdlib/3/http/client.pyi

```diff
@@ -150,7 +154,8 @@ class HTTPConnection:
         ) -> None: ...
     def request(self, method: str, url: str,
                 body: Optional[_DataType] = ...,
-                headers: Mapping[str, str] = ...) -> None: ...
+                headers: Mapping[str, str] = ...,
+                encode_chunked: bool = ...) -> None: ...
```

Comment:
Please mark as keyword-only argument.

```suggestion
                *, encode_chunked: bool = ...) -> None: ...
```

Also, since this argument was added in Python 3.6, there should be a version guard:

```python
if sys.version >= (3, 6):
    def request(self, method: str, url: str,
                body: Optional[_DataType] = ...,
                headers: Mapping[str, str] = ...,
                *, encode_chunked: bool = ...) -> None: ...
else:
    def request(self, method: str, url: str,
                body: Optional[_DataType] = ...,
                headers: Mapping[str, str] = ...) -> None: ...
```

Reply:
oh i missed the keyword only star and hadn't looked further back than 3.6. thanks!